### DISCUSSION TITLE
Asing for a data augementation for graph data.Now this time i design a two-stage network,too large for the first stage to do data augementation.

### DISCUSSION THREAD

**Discussion by fengsxy** [pyg-team/pytorch_geometric#4162] [Show and tell]
Hello i am new to pytorch_geometric.
Now this time i am trying use CNN get the feature first,and then output the data,but the output data is always large .For example the orginal image is about 100 KB and the output data is about 2.5MB.If i do the eight times data augementation,it will use a lot of disk space.So can we do the data augementation to the graph data.

**Comment by rusty1s** [comment]
What do you mean with outputting the data? Do you have a small code snippet/pseudo code to let me understand this? I don't think there is a reason to store augmentations on disk, you can just apply them on-the-fly, *e.g.*, via using `transforms` as illustrated [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/introduction.html#data-transforms).
